build/bin/build-client-binary

```diff
@@ -51,6 +51,12 @@ gobuild_component() {
         tar -C /tmp/go/src/${extract_path} --strip-components=1 -zxpf \${source_dir}/${component}-*.tar.gz
         pushd /tmp/go
             export GOPATH=\$(pwd -P)
+            if [ -d /tmp/go/src/github.com/percona/percona-toolkit ]; then
+                pushd /tmp/go/src/${extract_path}
+                    go get -u github.com/golang/dep/cmd/dep
```

Comment:
Please install tagged release: https://github.com/golang/dep/releases/tag/v0.5.4
Please also add a comment explaining why we add another ad-hoc step

Reply:
Let's wait for @percona-csalguero's response in Slack

Reply:
@EvgeniyPatlan Let's use tagged version there and below. We should not use `go get`.

Reply:
@AlekSi we use the same approach in pmm-1.x. so I just ported changes.

Reply:
So? We still should change that

Reply:
done